docs/content/docs/checks/AndroidLint.md

```diff
@@ -5,34 +5,42 @@ type: docs
 
 # Android Lint
 
-https://developer.android.com/studio/write/lint
+We use [Android lint](https://developer.android.com/studio/write/lint) to check applications in CI.
 
-В PR результаты отображаются во вкладках "\<app name\> lint".
+By default, we check only applications. It's no use to check libraries in isolation.
 
-## Настройки
+## How to run lint locally
 
-Помимо стандартных настроек lint у нас есть:
+`./gradlew :app:lintRelease`
```

Comment:
It is not clear what project command can works. 
Change `app` to `app-module-name` or mark page internal

Reply:
fixed